.travis.yml

```diff
@@ -3,7 +3,6 @@ os:
     - linux
     - osx
 julia:
-    - 0.3
```

Comment:
don't remove testing for it unless you also change the minimum version in REQUIRE. that's what actually matters to thr package manager